README.md

```diff
@@ -432,8 +432,7 @@ See the [Phone dial-out](#phone-dial-out) section.
 
 * `USE_GOOGLE_FONTS` (Optional, default value: true): Whether the client app will load
```

Comment:
Please be aware that the USE_GOOGLE_FONTS property is listed twice in the readme. One should go away :)

Reply:
Also, this env tracks down to the `useGoogleFonts` variable, which i see being referenced but not used. Am i missing something?